README.rst

```diff
@@ -32,11 +32,11 @@ Listed below are the actively maintained release branches along with their lates
 minor release, corresponding image pull tags and their release notes:
 
 +---------------------------------------------------------+------------+-----------------------------------+---------------------------------------------------------------------------+--------------------------------------------------------------------------------+
-| `v1.11 <https://github.com/cilium/cilium/tree/v1.11>`__ | 2021-12-09 | ``quay.io/cilium/cilium:v1.11.0`` | `Release Notes <https://github.com/cilium/cilium/releases/tag/v1.11.0>`__ | `General Announcement <https://isovalent.com/blog/post/2021-12-release-111>`__ |
+| `v1.11 <https://github.com/cilium/cilium/tree/v1.11>`__ | 2021-12-09 | ``quay.io/cilium/cilium:v1.11.1`` | `Release Notes <https://github.com/cilium/cilium/releases/tag/v1.11.1>`__ | `General Announcement <https://isovalent.com/blog/post/2021-12-release-111>`__ |
```

Comment:
What do these dates refer to?

Reply:
I believe it's meant to be the release date of the current point release for the series, but I agree this is not obvious and also somehow we seem to have broken the script, because only the 1.10.7 version date was updated here.

Reply:
There doesn't seem to be any recent changes to the script that could be related to this, I think it's just that the implementation caught up with us. Version strings have `.` in them, and we use the version strings to pass into `sed` expressions so it's likely that we have just constructed the substitution logic poorly.

Reply:
OK, two problems:
* @borkmann manually updated the v1.11.x release date last time, so it didn't match the git logs, so when the script next ran, it couldn't find the 'old date' to update it inline.
* the version match string was inadvertently matching one of the dates, because it was just `1.11.0` rather than `v1.11.0`. `1.11.0` regex matches `2021-11-05`.

Reply:
I have a couple of local fixes to improve this, will send a separate PR.